system/handlers/Tasks.cfc

```diff
@@ -18,4 +19,17 @@ component {
 	private boolean function processBatchedEmails( logger ) {
 		return emailMassSendingService.processQueue( arguments.logger ?: NullValue() );
 	}
+
```

Comment:
I'm not sure that we should send this out on a schedule. I think more preferable would be to resend only when they attempt to reset their password with an expired token.